packages/react-core/src/components/Pagination/Pagination.tsx

```diff
@@ -141,6 +141,8 @@ export interface PaginationProps extends React.HTMLProps<HTMLDivElement>, OUIAPr
   onPageInput?: (event: React.SyntheticEvent<HTMLButtonElement>, page: number) => void;
   /** Function called when user selects number of items per page. */
   onPerPageSelect?: OnPerPageSelect;
+  /** Component to be used for wrapping the toggle contents. */
+  component?: 'div' | 'button';
```

Comment:
I think this would be called something more specific to the menu, like `perPageComponent` or something like that. Otherwise it seems like it applies to the `<Pagination>`'s `<div class="pf-c-pagination">` element. It would also add a namespace conflict if we created a `component` prop for `div.pf-c-pagination`